data/dbus-interfaces/com.endlessm.HackableApp.xml

```diff
@@ -1,5 +1,8 @@
 <node>
   <interface name="com.endlessm.HackableApp">
+    <method name="PulseFtH">
```

Comment:
I have other suggestions for the method name: "PulseFlipButton" or "PulseFlipToHackButton". Since "FtH" is a shortcut for the "lfip to hack" feature, but this method is about pulsating the button. In case you accept my suggestions, the clubhouse part should be adapted too.

Reply:
I agree, it's better to be more explicit, I've done that change and I'll update the clubhouse part too.